src/app/components/split-view/index.html

```diff
@@ -0,0 +1,229 @@
+<sky-tabbed-demo-page
+  pageTitle="Split view"
+>
+  <sky-demo-page-summary>
+    <p>
+      The split view component allows users to work through a list of items, while displaying each item's details in context. The split view component consists of two children components: the drawer and the workspace.
+    </p>
+  </sky-demo-page-summary>
+
+  <sky-demo-page-example>
+    <sky-split-view-demo></sky-split-view-demo>
+  </sky-demo-page-example>
+
+<sky-tabset id="content-tabs">
+
+  <sky-tab
+    tabHeading="Design">
+
+    <sky-demo-page-content sectionHeading="Usage">
+      <h3 class="sky-text-success">Use when</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Use a split view if a user needs to work through a list of items with the intent of taking an action on each item in the list.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <img src="~/assets/img/guidelines/split-view/use-when.png" />
```

Comment:
@Blackbaud-ToddRoberts, can we resize this image so that it fits alongside the text? I think it would still be readable at that size. ... Also, shouldn't this image include a caption? I think that's the pattern for images that illustrate when to use or not use components, so it seems like it should have a caption here. Plus, the caption could call out that in this example, checking people in is the action, which would be good because I think it's easy to overlook the check in button.

Reply:
Actually, can we just make a new, narrower image instead of resizing the existing one? The images that follow this aren't as wide. Is there some reason we need a wider image here?

Reply:
the intent was to use a somewhat realistic width for these full-page-context examples, but could shrink to e.g. 800px. Whether it fits next to the text or not is happenstance based on monitor size - on my screen the usage example is side by side.

Reply:
Is the realistic width for full-page context a new thing? I don't see that pattern in the other docs we've updated. Phone field and tabs don't user larger full-page-context examples in the Usage section, and popover doesn't have any images in the Usage section. ... Anyway, I just thought this looked a little weird since all the other images work side-by-side for me except this one. And my browser is at full size on a fairly wide screen, so I think this will be the case for most users. No big deal if that's how we want to handle this, but I figured it was worth asking. Thanks.

Reply:
just pushed an update with narrower images

Reply:
Sounds good.